lib/spec/runners/operations.ex

```diff
@@ -43,19 +43,28 @@ defmodule OperationsTestRunner do
   @disabled_handlers [
     # "attester_slashing",
     # "attestation",
-    # "deposit",
     # "block_header",
+    # "deposit",
     # "proposer_slashing",
     # "voluntary_exit",
     # "sync_aggregate",
-    "execution_payload"
+    # "execution_payload"
     # "withdrawals",
     # "bls_to_execution_change"
   ]
 
+  @disabled_cases [
+    "bad_parent_hash_first_payload"
```

Comment:
I will fix this case in a follow up since it requires an update to the spec test version and might come with several other changes.